docs/contracts/functions.rst

```diff
@@ -41,6 +41,10 @@ that call them, similar to internal library functions.
     is that free functions do not have direct access to the variable ``this``, storage variables and functions
     not in their scope.
 
+There is a special category of free functions called :ref:`literal suffixes<literal_suffixes>`.
+Such functions are marked with the ``suffix`` modifier, and can be used to define convenient
+conversions from literals to arbitrary types.
```

Comment:
```suggestion
There is a particular category of free functions called :ref:`literal suffixes<literal_suffixes>`.
Such functions are marked with the ``suffix`` modifier and can be used to define convenient
conversions from literals to arbitrary types.
```

Reply:
Yeah, that was my thinking behind it. I also thought `special` made more sense so maybe I'll change it back.

Reply:
Changed back to `special`.